### DISCUSSION TITLE
Determining min_area and what do the numbers next to the bounding boxes mean

### DISCUSSION THREAD

**Discussion by zaPure** [blakeblackshear/frigate#1991] [Ask A Question] [answered]
Hi 

Just started using frigate and wanted to stop my small dog from being detected as a person by setting a min_area.
According to the configuration file page in the documentation. 
min_area = minimum width*height of the bounding box for the detected object 
But how do I work out the width and height of the bounding box around my dog using the last 5 events? Is the information stored anywhere?

I was also curious what does the number on the top right of the orange bounding box mean? I went through the 15 pages of the discussions and couldn't find any reference to it.

**Comment by jakowenko** [comment, marked as answer]
The number in the orange bounding box should be the area for that box.

**Reply by zaPure** [reply]
thank you, will give it go
